charts/datasaur/templates/config/datasaur-env-configMap.yaml

```diff
@@ -12,6 +12,18 @@ data:
   LANGUAGE_TOOL_SERVICE_ENDPOINT: http://{{ .Release.Name }}-language-tool-service:8010/v2/check
   PRESIDIO_ANALYZER_ENDPOINT: http://{{ .Release.Name }}-presidio-analyzer-service:3000/analyze
   WAVEFORM_PEAKS_GENERATOR_HOST: http://{{ .Release.Name }}-audiowaveform-generator-service:5005
+  URL_INSPECTOR_INGRESS_URL: https://{{ .Values.ingress.host }}
+  URL_INSPECTOR_INGRESS_HOST: {{ .Values.ingress.host }}
+{{- if .Values.services.urlInspector.proxy }}
+  ENABLE_PROJECT_CREATION_WITH_PROXY: "true"
+  SESSION_DOMAIN: {{ .Values.ingress.host }}
+  UPLOAD_FILE_PROXY_URL: {{ .Values.ingress.host }}/api/static/proxy/upload
+  MAX_TOKENS_PER_SENTENCE: "2000"
```

Comment:
oh ya, i think env var MAX_TOKENS_PER_SENTENCE is not specific for upload-proxy, so you can move it out from it